src/setup/i18n.ts

```diff
@@ -42,20 +38,18 @@ export const i18n = new VueI18n({
   numberFormats,
 });
 
+export default i18n;
+
 /**
  * Load messages for given locale if not already loaded.
- *
- * @param   {String}    locale    A locale identifier
- *
- * @returns {Promise}
  */
-export const i18nLoadMessages = function(locale) {
-  if (!Object.keys(i18n.messages).includes(locale)) {
+export const i18nLoadMessages = function(locale: string) {
+  if (!Object.keys(i18n.global.messages).includes(locale)) {
     return import(/* webpackChunkName: 'lang-[request]' */`../translations/${locale}`)
-      .then((localeMessages) => {
+      .then(({ default: localeMessages }) => {
```

Comment:
Hmmm... This looks wrong to me. I think `import()` should already return the `default` export. Why is this not working anymore?